lib/apiservers/portlayer/restapi/handlers/interaction_handlers.go

```diff
@@ -238,8 +291,9 @@ func (d *FlushingReader) AddFlusher(flusher GenericFlusher) {
 func (d *FlushingReader) WriteTo(w io.Writer) (written int64, err error) {
 	buf := make([]byte, 64)
 
+	nr, er := d.readDetectInit(buf)
```

Comment:
WriteTo() is only called once for the stream.  io.copy() only calls WriteTo() once.